static/styles/lib/chatLoggedin.scss

```diff
@@ -0,0 +1,232 @@
+			background-size: 36px 36px;
+			width: 36px; /* Width of new image */
+			height: 36px; /* Height of new image */
+			padding-left: 36px; /* Equal to width of new image */
+		}
+
+
+		.mx_TopLeftMenuButton {
+			background-color: transparent;
+			color: $colorWhite;
+			border: none;
+		}
+
+		.mx_TopLeftMenuButton_name {
+			font-size: 20px;
+			//font-size: var(--text-lg);
+		}
+
+		.my_CollapseButtons {
+			background-color: transparent;
+			border: none;
+		}
+
+		.mx_RightPanel_headerButton:before {
+			background-color: $colorWhite;
+		}
+	}
+
+	// > Menu Content
+	.mx_RoomSubList_scroll {
+		padding: 0;
+	}
+
+	.mx_RoomTile {
+		padding: 8px 14px;
+		height: 24px;
+	}
+
+	.mx_RoomTile:hover {
+		background-color: #E3E7E9; // var(--color-tertiary) => 15%;
+	}
+
+	.mx_RoomTile_selected {
+		background-color: #E3E7E9; // var(--color-tertiary) => 15%;
```

Comment:
will this work in all themes?

Reply:
It's only a light grey, so it should work in all themes, but anyway `$colorGrey` might be a better way to do it so we can adjust it more easaly.

Reply:
Fixed by calculating the color based on a variable.